docs/source/getting_started.rst

```diff
@@ -37,7 +37,7 @@ Building from source
 --------------------
 
 ``numba-dpex`` can be built from source using either ``conda-build`` or
-``setuptools``.
+``scikit-build``.
```

Comment:
The build step is still using setuptools as you are doing `python setup.py develop`. scikit-build is hidden from end users.

Reply:
Fixed.